### PR TITLE
add 111

### PR DESCRIPTION
**Pull request by ShidieRen**
This changes add 111

### CODE REVIEW

File: src/components/auth/SignIn.tsx [hunk 1]
```diff
@@ -64,7 +64,7 @@ function SignIn() {
           <div>
             <GoogleButton
               style={{ display: "block", margin: "auto" }}
-              // type="light"
+              // type="light1111"
```
Comment (guanhuali1018) [inline src/components/auth/SignIn.tsx:67]: what's this?

Comment (ShidieRen) [reply src/components/auth/SignIn.tsx:67]: 1112

**Comment by guanhuali1018** [comment]
Looks good to me!

**Comment by guanhuali1018** [comment]
dsfffdsfsd

**Review by guanhuali1018 — APPROVED** [review]
great!!

**Pull request merged**